tgui/packages/tgui/components/DepartmentPane.tsx

```diff
@@ -73,8 +76,8 @@ export const DepartmentPane: SFC<{
                 data.titleSubtextBuilder && data.titleSubtextBuilder(department)
               }>
               <Stack vertical>
-                {department.jobs.map((job) =>
-                  data.jobEntryBuilder(job, department)
+                {Object.entries(department.jobs).map((job) =>
+                  data.jobEntryBuilder(job[0], job[1], department)
```

Comment:
I suggest wrapping each entry with `<Stack.Item>` here, instead of delegating it to `jobEntryBuilder`, and I suggest renaming this prop into `renderJobEntry`, to reduce ambiguity, it renders an element on the page.